src/index.html

```diff
@@ -12,108 +12,123 @@
         rel="stylesheet">
     <link href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" rel="stylesheet" />
     <!--style reference from node_modules/@syncfusion/ej2/-->
-    <link href="/styles/styles.css" rel="stylesheet" />
+    
 
-    <script src="node_modules/systemjs/dist/system.src.js" type="text/javascript"></script>
+    <script src="/node_modules/systemjs/dist/system.src.js" type="text/javascript"></script>
```

Comment:
why you have added /

Reply:
It is added, so that the file is referenced correctly